libfreerdp/core/server.c

```diff
@@ -944,6 +966,13 @@ HANDLE WINAPI FreeRDP_WTSVirtualChannelOpen(HANDLE hServer, DWORD SessionId, LPS
 	hChannelHandle = (HANDLE) channel;
 
 	return hChannelHandle;
+
+error_queue:
+	Stream_Free(channel->receiveData, TRUE);
+error_receiveData:
+	free(channel);
```

Comment:
It is correct here but I see a little risk here because the memory pointed to by channel is not allocated here and if future error_xx gotos are added it could easily be overseen. If you don't think that this is too paranoid I'd suggest to assign channel to NULL in the declaration, directly assign hChannelHandle in line 945 and move line 966 after line 963 .... or whatever fixes the risk ;)